src/index.html

```diff
@@ -4,9 +4,311 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Gadgets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Tablets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Photo</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
I noticed that the displayed products are quite repetitive. To make the interface more engaging, I suggest creating greater variety in the sample items.

You can achieve this by iterating over an array of objects containing different product information, such as names, prices, and images. This can be easily combined with the previously mentioned rendering loop. This solution allows you to display a dynamic list of items with good variety while reducing the effort of manual replication.